<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NEOLAP000004338" failures="16" tests="34" name="EmployeesRestAPITest" time="25.173" errors="0" timestamp="2022-09-10T00:49:56 IST">
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="@BeforeClass deleteEmployee" time="0.998">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.employeeapi.testCases.TC005_Delete_Employee_Record.deleteEmployee(TC005_Delete_Employee_Record.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:46)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:50)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 30 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass deleteEmployee -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="@AfterClass tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkContentLenght" time="0.01">
    <failure type="java.lang.NumberFormatException" message="Cannot parse null string">
      <![CDATA[java.lang.NumberFormatException: Cannot parse null string
at java.base/java.lang.Integer.parseInt(Integer.java:630)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at com.employeeapi.testCases.TC001_Get_All_Employees.checkContentLenght(TC001_Get_All_Employees.java:121)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkContentLenght -->
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkContentType" time="0.015"/>
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkCookies" time="0.007"/>
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkResponseTime" time="0.008"/>
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkResposeBody" time="0.043"/>
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkStatusCode" time="0.023">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkStatusCode(TC001_Get_All_Employees.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkcontentEncoding" time="0.006"/>
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkserverType" time="0.012">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkserverType(TC001_Get_All_Employees.java:100)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.employeeapi.testCases.TC001_Get_All_Employees" name="checkstatusLine" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkstatusLine(TC001_Get_All_Employees.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkContentLenght" time="0.003">
    <failure type="java.lang.NumberFormatException" message="Cannot parse null string">
      <![CDATA[java.lang.NumberFormatException: Cannot parse null string
at java.base/java.lang.Integer.parseInt(Integer.java:630)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkContentLenght(TC002_Get_Single_Employee_Record.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkContentLenght -->
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkContentType" time="0.011"/>
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkResponseTime" time="0.002"/>
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkResposeBody" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkResposeBody(TC002_Get_Single_Employee_Record.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkStatusCode" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkStatusCode(TC002_Get_Single_Employee_Record.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkserverType" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkserverType(TC002_Get_Single_Employee_Record.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record" name="checkstatusLine" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkstatusLine(TC002_Get_Single_Employee_Record.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkContentType" time="0.003"/>
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkResposeBody" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkResposeBody(TC003_Post_Employee_Record.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkStatusCode" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkStatusCode(TC003_Post_Employee_Record.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkcontentEncoding" time="0.002"/>
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkserverType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [Apache]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [Apache]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkserverType(TC003_Post_Employee_Record.java:95)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.employeeapi.testCases.TC003_Post_Employee_Record" name="checkstatusLine" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkstatusLine(TC003_Post_Employee_Record.java:82)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkContentType" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkContentType(TC004_Put_Employee_Record.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkResposeBody" time="0.012"/>
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkStatusCode" time="0.006"/>
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkcontentEncoding" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkcontentEncoding(TC004_Put_Employee_Record.java:101)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkserverType" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkserverType(TC004_Put_Employee_Record.java:95)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.employeeapi.testCases.TC004_Put_Employee_Record" name="checkstatusLine" time="0.003"/>
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkContentType" time="0.001">
    <skipped/>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkResposeBody" time="0.0">
    <skipped/>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkStatusCode" time="0.0">
    <skipped/>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkcontentEncoding" time="0.0">
    <skipped/>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkserverType" time="0.001">
    <skipped/>
  </testcase> <!-- checkserverType -->
  <testcase classname="com.employeeapi.testCases.TC005_Delete_Employee_Record" name="checkstatusLine" time="0.0">
    <skipped/>
  </testcase> <!-- checkstatusLine -->
</testsuite> <!-- EmployeesRestAPITest -->
